Makefile

```diff
@@ -64,5 +64,11 @@ test_crictl:
 	-@sudo GOFLAGS="-count=1" $$(which go) test ./tests/crictl -v
 	@echo " "
 
+test_unikontainers:
+	@GOFLAGS="-count=1" $$(which go) test ./pkg/unikontainers -v
+
+unittest: test_unikontainers
+	-@echo "unit tests: DONE"
+
 test: test_nerdctl test_ctr test_crictl
```

Comment:
We should include unit testing in `test` too.

Reply:
Yes, that's correct. I included it.